<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptsDialog</class>
 <widget class="QWidget" name="ReceiptsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>498</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Receipts</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_9"/>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditName">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="labelFirstname">
         <property name="text">
          <string>Firstname</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>lineEditFirstname</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEditFirstname">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLineEdit" name="lineEditFreeValue"/>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="labelFreeValue">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEditFreeName"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelFreeName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>lineEditName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxFreeEntry">
         <property name="text">
          <string>Free entry</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxFreeValue">
         <property name="text">
          <string>Free value</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>300</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Values</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>17</height>
                  </size>
                 </property>
                 <property name="text">
                  <string> Date of payment</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="buddy">
                  <cstring>dateTimeEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDateTimeEdit" name="dateTimeEdit">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>27</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Date of action</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="buddy">
                  <cstring>dateTimeEdit_2</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Debtor</string>
                 </property>
                 <property name="buddy">
                  <cstring>comboBoxDebtor</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="comboBoxDebtor"/>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0">
                <widget class="QLabel" name="labelThesaurus">
                 <property name="text">
                  <string>Thesaurus</string>
                 </property>
                 <property name="buddy">
                  <cstring>comboBoxThesaurus</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboBoxThesaurus"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Distance</string>
                 </property>
                 <property name="buddy">
                  <cstring>spinBox</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="spinBox"/>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="receiptsFindButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Find</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelReceiptsLists">
                 <property name="text">
                  <string>Receipts lists</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Where and who</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>261</width>
             <height>241</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Where</string>
              </property>
              <property name="buddy">
               <cstring>comboBoxWhere</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxWhere"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Who</string>
              </property>
              <property name="buddy">
               <cstring>comboBoxWho</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxWho"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Bank Account</string>
              </property>
              <property name="buddy">
               <cstring>comboBoxBankAccount</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxBankAccount"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Rules</string>
              </property>
              <property name="buddy">
               <cstring>comboBoxRules</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="comboBoxRules"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_14">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>17</height>
               </size>
              </property>
              <property name="text">
               <string>Distance rule</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>comboBoxDistance</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="comboBoxDistance"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>65</height>
          </size>
         </property>
         <property name="title">
          <string>Percents</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>261</width>
            <height>58</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="percentLabel">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="lessButton">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="percentBar">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="plusButton">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QRadioButton" name="cashRadioButton">
           <property name="text">
            <string>Cash</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="checkRadioButton">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="cardRadioButton">
           <property name="text">
            <string>Credit card</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="bankRadioButton">
           <property name="text">
            <string>Banking</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="dueRadioButton">
           <property name="text">
            <string>Dues</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="saveButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>90</height>
            </size>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Sequences</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="showButton">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>180</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="registerLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>300</width>
            <height>170</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>170</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_10"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
